src/data/weather.ts

```diff
@@ -22,15 +27,15 @@ export class Weather {
 
   constructor(weatherType: WeatherType, turnsLeft?: number) {
```

Comment:
```suggestion
  constructor(weatherType: WeatherType, turnsLeft: number = 0) {
```
Huh. Well, since your changes revealed this might as well fix it lol.